dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/OfficialName/Confirm.cshtml.cs

```diff
@@ -38,11 +47,25 @@ public void OnGet()
     {
```

Comment:
Not used:
    [FromQuery(Name = "fileId")]
    public string? FileId { get; set; }